docs/src/app/components/pages/components/DatePicker/Page.jsx

```diff
@@ -59,6 +62,13 @@ const DatePickerPage = () => (
       <DatePickerExampleControlled/>
     </CodeExample>
     <CodeExample
+      title="Disabled dates example"
+      description={descriptions.disabledDates}
+      code={datePickerExampleDisableDatesCode}
+    >
+      <DatePickerExampleDisableDates/>
```

Comment:
Ah, that would explain it! Thought I'd been picked up on this before - must have been something else.